packages/inventory-compliance/src/SystemRulesTable/SystemRulesTable.js

```diff
@@ -29,8 +29,11 @@ export const selectColumns = (columnTitles) => (
 );
 
 class SystemRulesTable extends React.Component {
+    remediationAvailableFilter = this.props.remediationAvailableFilter || this.props.remediationsEnabled;
```

Comment:
@vkrizan Updated. I'm not sure this is the 'right' way to do this, but it seems to work OK.

Reply:
:+1: